### ISSUE TITLE
Unable to install plugin

### ISSUE THREAD

**Issue by jfconavarrete** [logstash-plugins/logstash-input-jms#17]
I am being unable to install the latest version of the plugin using logstash version 5.2.2, I'm getting the following error message:

```
$ bin/logstash-plugin install logstash-input-jms
Validating logstash-input-jms
Installing logstash-input-jms
Error Bundler::InstallError, retrying 1/10
An error occurred while installing semantic_logger (4.0.0), and Bundler cannot continue.
Make sure that `gem install semantic_logger -v '4.0.0'` succeeds before bundling.
...
...
Too many retries, aborting, caused by Bundler::InstallError
ERROR: Installation Aborted, message: An error occurred while installing semantic_logger (4.0.0), and Bundler cannot continue.
Make sure that `gem install semantic_logger -v '4.0.0'` succeeds before bundling.
```
As the command said, I tried running $gem install semantic_logger -v '4.0.0' and it worked correctly.

I have MacOS Sierra 10.12
```
$ bin/logstash --version
logstash 5.2.2
```

Any ideas about how to fix this?
Thanks

**Comment by AgentW4C** [comment]
Hmm, just wondering where the semantic_logger got installed in your case @jfconavarrete. I have tried that  on my machine and got to the error:
```
ERROR:  Error installing semantic_logger:
        semantic_logger requires Ruby version >= 2.1.
```

**Comment by jfconavarrete** [comment]
I was getting that error too so I upgraded my system's Ruby version to be 2.2 using RVM

**Comment by ranierimazili** [comment]
Same problem here.

I'm trying to install logstash-input-jms plugin but when I run the command logstash-plugin install --no-verify logstash-input-jms I'm receiving the following message:

```
Installing logstash-input-jms
Error Bundler::InstallError, retrying 1/10
An error occurred while installing semantic_logger (4.0.0), and Bundler cannot continue.
ERROR: Installation Aborted, message: An error occurred while installing semantic_logger (4.0.0), and Bundler cannot continue.
Make sure that `gem install semantic_logger -v '4.0.0'` succeeds before bundling.
```

So, investigating the problem I've try to execute the command mentioned in the error using gem that inside logstash (vendor/jruby/bin/gem) and I've received the following message when I tried to run the command to install semantic_logger

```
gem install semantic_logger -v '4.0.0'
Fetching: concurrent-ruby-1.0.5-java.gem (100%)
Successfully installed concurrent-ruby-1.0.5-java
Fetching: semantic_logger-4.0.0.gem (100%)
ERROR:  Error installing semantic_logger:
        semantic_logger requires Ruby version >= 2.1.
```

I could see that logstash-5.2.2 uses jruby 1.7.25.
```
jruby.bat --version
jruby 1.7.25 (1.9.3p551) 2016-04-13 867cb81 on Java HotSpot(TM) 64-Bit Server VM 1.8.0_121-b13 +jit [Windows 10-amd64]
```

One week ago I could install this plugin without problems but now I can't. I also noticed that a new version of semantic_logger (4.0.0) was released two days ago, probably something changed and that's the reason of the problem but I have no idea of how to solve it.

Does anybody knows how to make it work?

Thanks

**Comment by ranierimazili** [comment]
Other findings... don't know if this is possible but force to use semantic_logger 3.4.1 instead of the new version (4.0.0) could solve the problem.
I've tried to upgrade logstash jruby version and it doesn't work, probably they change something inside.

**Comment by jordansissel** [comment]
Sounds like a bug introduced by a dependency for this plugin. Fixing this
will likely require a patch to this plugin.

A workaround maybe possible but I am on my phone and can't try right now.

**Comment by AgentW4C** [comment]
Issue fixed in the plugin release v3.0.1
